src/app/map/map.component.ts

```diff
@@ -973,65 +1006,141 @@ export class MapComponent implements OnInit {
             );
             //If the user has at least one Event, Network, Sensor, or State filter select, continue with http request
         } else {
-            // format url params into single string
-            let urlParamString =
-                'Event=' +
-                eventId +
-                '&State=' +
-                stateAbbrevs +
-                '&SensorType=' +
-                sensorIds +
-                '&NetworkName=' +
-                networkIds +
-                '&OPDefined=' +
-                opDefinedTrue +
-                '&HWMOnly=' +
-                HWMTrue +
-                '&HWMSurveyed=' +
-                surveyed +
-                '&SensorOnly=' +
-                sensorTrue +
-                '&RDGOnly=' +
-                RDGTrue +
-                '&HousingTypeOne=' +
-                bracketTrue;
-
-            //Clear current markers when a new filter is submitted
-            if (this.map.hasLayer(this.siteService.siteMarkers)) {
-                this.siteService.siteMarkers.clearLayers();
-            }
-            //Find sites that match the user's query
-            this.siteService
-                .getFilteredSites(urlParamString)
-                .subscribe((res) => {
-                    //set the state control back to state names instead of abbreviations
-                    this.mapFilterForm
-                        .get('stateControl')
-                        .setValue(this.selectedStates);
-                    //only call mapResults if the query returns data
-                    if (res.length > 0) {
-                        //close the filter panel
-                        this.filtersPanelState = false;
-                        this.filtersService.updateSites(res);
-                        this.mapResults(
-                            res,
-                            this.eventIcon,
-                            this.siteService.siteMarkers,
-                            true
-                        );
-
-                        // updating the filter-results table datasource with the new results
-                        this.filterResultsComponent.refreshDataSource();
-                    } else {
-                        //if nothing is returned, show a snack bar message
-                        this.filtersSnackBar(
-                            'No results for your query. Try using fewer filters.',
-                            'OK',
-                            4500
-                        );
+            //network ids need to be called one at a time, so if there are multiple we put them in separate http requests
+            if (multiNetworkIds.length <= 1) {
+                // format url params into single string
+                let urlSingNetwork =
```

Comment:
Should this variable be called `urlStringNetwork`?

Reply:
it was short for single (instead of multi). It didn't end up being necessary that those two variables have different names, so I can just change them back to the original